CHANGES

```diff
@@ -4,7 +4,41 @@ Release 2.0.0 (in development)
 Dependencies
 ------------
 
-* LaTeX builder now depends on TeX Live 2015 or above
+* LaTeX builder now depends on TeX Live 2015 or above.
+* LaTeX builder may need these additional LaTeX packages for PDF builds
+  (with ``'pdflatex'`` :confval:`latex_engine`):
+
+  .. list-table:: Requirements
+     :header-rows: 1
+
+     * - LaTeX package (CTAN)
+       - Ubuntu xenial
+       - needed for
+     * - substitutefont
+       - texlive-latex-extra
+       - Greek or Cyrillic letters (in non-Cyrillic documents)
+     * - textalpha
+       - texlive-lang-greek
+       - Greek letters (in text, not math)
+     * - gfsartemisia
+       - fonts-gfs-artemisia (texlive-fonts-extra)
+       - Greek letters (in text, not math)
+     * - gfsneohellenic
+       - fonts-gfs-neohellenic (texlive-fonts-extra)
+       - Greek letters (in text, not math)
+     * - cbfonts
+       - texlive-lang-greek
+       - Greek letters (in text, not math)
+     * - cm-lgc
+       - texlive-fonts-extra
+       - Cyrillic letters (in non-Cyrillic documents)
+
+  These extra package are not required by default. The first two are needed if
+  the :confval:`latex_elements`.\ ``'fontenc'`` key has been modify to declare
+  the use of the ``LGR`` (Greek) and/or ``T2A`` (Cyrillic) font encoding. Even
+  then, the last four are font packages arising in the default value for
+  :confval:`latex_elements`.\ ``'fontpkg'``, and may be replaced by other font
+  packages providing ``LGR`` and/or ``T2A`` support.
```

Comment:
These explanations may move later on to builders/index.rst from CHANGES, after 2.0 release